web-components/src/components/cards/ProjectCard/ProjectCard.tsx

```diff
@@ -231,8 +233,9 @@ export function ProjectCard({
                       track('buy1', {
                         url: location.pathname,
                         cardType: 'project',
-                        creditClassName: purchaseInfo.creditClass?.name,
```

Comment:
this was not working properly so i've removed it, there is a separate to task to further clean up the extra metadata we are sending for these events, so we can fix this at a later time. it's not a critical field to have right now based on my convo with @erikalogie earlier today.